### PR TITLE
LPD-29643 Functional test for web content structure filter

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Created functional test to cover LPP-50468 After clicking on Clear (filter and order) you can still see the results of the filter

# What is this trying to solve?
[Link to ticket](https://liferay.atlassian.net/browse/LPD-29643)

The LPP https://liferay.atlassian.net/browse/LPP-50468 solved an issue when filter by Structure in Web Content. With this test we check that the search is correctly cleared after click in Clear button. 

**Steps to reproduce the issue:**
1. Create a structure with a text field
2. Create some basic web content without the structure
3. Create some basic web content using the structure
4. Filter by structure and see that the results are correct
5. Click on clear the search results

# How can you verify that it works?
Run `npm run test -- -g "LPP-50468" --reporter list `

Screenshot 2024-07-02 at 15 37 38

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:playwright

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:report:28037121

**Comment by ambrinchaudhary** [comment]
> **Unique Failures**
> journal-web/journalPage.spec.ts,https://testray.liferay.com/#/project/34754/routines/34756/build/28037121/case-result/28037760/history,Unique,LPP-50468 After clicking on Clear (filter by structrure) you can see all the web contents: journalPage.spec.ts:22:1 LPP-50468 After clicking on Clear (filter by structrure) you can see all the web contents

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:report:29486855

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:playwright

**Comment by ambrinchaudhary** [comment]
New playwright test passed successfully on CI

Screenshot 2024-07-03 at 11 45 00

File: modules/test/playwright/tests/journal-web/journalPage.spec.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	});
+
+	const structure = await apiHelpers.dataEngine.createStructure(site.id, dataDefinition);
+
+	//3) Create web content based on structure
+	await apiHelpers.jsonWebServicesJournal.addWebContent({
+		ddmStructureId: structure.id,
+		groupId: site.id,
+		titleMap: {en_US: 'Second Web Content'}
+	});
```
Comment (boton) [inline modules/test/playwright/tests/journal-web/journalPage.spec.ts]: I suggest moving into `/utils/` function and avoiding comments. 

Something like [blogs-web/blogEntry.spec.ts](https://github.com/liferay/liferay-portal/blob/d66cc464ba0724b676b62463bd2b2a28a6d9b402/modules/test/playwright/tests/blogs-web/blogEntry.spec.ts#L81-L89) and [blogs-web/utils/](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/blogs-web/utils/).

But I think you can do it all in one file ❤️

**Comment by boton** [comment]
🥳 
image

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**